pkgs/development/python-modules/pybind11/default.nix

```diff
@@ -0,0 +1,28 @@
+{ stdenv, python }:
+
+with python.pkgs;
+
+buildPythonPackage rec {
+  pname = "pybind11";
+  version = "2.2.4";
+
+  src = fetchPypi {
+    inherit pname version;
+    sha256 = "1kz1z2cg3q901q9spkdhksmcfiskaghzmbb9ivr5mva856yvnak4";
+  };
+
+  doCheck = false;
```

Comment:
Why?

Reply:
There is not test suite in the python case.

Reply:
Guess you were asking for a comment explaining this, not me to remove it. Will add it back with comment.